ani-cli

```diff
@@ -525,6 +530,9 @@ while getopts 'svq:dp:chDUVa:xr:fn' OPT; do
 		p)
 			player_fn='download'
 			download_dir="$OPTARG"
+			if [ ! -d "$download_dir" ] ; then
+				die "Directory \"$download_dir\" does not exist."
+			fi
```

Comment:
It's sufficient to implement your fix either here or in the download() function. 
I'd prefer it if you implement it here